docs/docsite/rst/playbook_guide/playbooks_delegation.rst

```diff
@@ -13,7 +13,7 @@ Tasks that cannot be delegated
 
 Some tasks always execute on the control node. These tasks, including ``include``, ``add_host``, and ``debug``, cannot be delegated.
 You can determine if an action can be delegated from the ``connection`` attribute documentation.
-If the ``connection`` attribute indicates ``support`` is ``False`` or ``None``, then the action does not use a connection and cannot be delegated.
+If the ``connection`` attribute indicates ``support`` is ``true`` or ``false``, then the action does not use a connection and cannot be delegated.
```

Comment:
I don't think this has the same meaning. The original equates False with None. This change changes false to true. My nickel is the change required here may only be to change False to false, but let's see what @bcoca thinks....

Reply:
If this is talking about the Python code which uses `True` (capital `T`) and `None`, it should stay as is.

Reply:
good point. If I dig deeper, the 'connection' attribute might be listed in our attributes tables such as https://docs.ansible.com/ansible-core/2.14/collections/ansible/builtin/add_host_module.html#attributes  which... thickens the plot, because it uses `none` aka lowercase there... @felixfontein might remember but if it's referring to  those tables, then I don't think False/false is even an option....

Reply:
@gotmax23 and @samccann Thanks for taking a look. You're right that this shouldn't change. I was a bit anxious that I'd merged the other PR too soon after @briantist pointed out the `true` / `false` thing and wanted to follow up. Probably should have just referred to the docs in the matrix channel rather than opening this PR.

With the lowercase thing, my understanding is that `None` means something different to `none`. The former is a constant and represents a null value while `none` needs to be defined somewhere. That said I could be overthinking it and this is just for the `supported` field in the documentation and not a value that users need to set somewhere.

We should probably close this. I think I'm just making noise...

Reply:
In the case of attributes, `True`/`true` and `None`/`null` are equivalent and for display only (for now), I wrote them capitalized to emphasize the values more than anything else.  In any case, the change from `True`/`None` to `true`/`false` is incorrect, at best it would be `true`/`null`.